### ISSUE TITLE
Move listView().addQuickFiter() to the filterView

### ISSUE THREAD

**Issue by fzaninotto** [marmelab/ng-admin#194]
Doesn't make sense to keep it under the list view anymore now that we have a dedicated view for filters.

**Comment by fzaninotto** [comment]
In fact, after using it, the `filtersView()` feels a bit clunky. If main views are splittable, why is there a `filtersView` and not a `datagridView`? Also, too many views complexifies documentation. 

I think the config API would be easier to understand and use like the following:

```js
myEntity.listView()
  .columns([
    new Field('foo'),
    new Field('bar')
  ])
  .filters([
    new Field('foo'),
    new Field('bar')
  ])
  .quickFilters([
    new Field('foo'),
    new Field('bar')
   ]);

myEntity.editionView()
  .fields([
    new Field('foo'),
    new Field('bar')
  ]);
```

So we should not move the quickFilters into FiltersView, but rather merge the filtersView with the listView.

Sorry for the back-and forth on the config API, but it' only by using it that the weaknesses appear. Until this is implemented, please refrain from using filters.

**Comment by manuquentin** [comment]
Fixed in #223
